dolphinscheduler-extract/dolphinscheduler-extract-base/src/main/java/org/apache/dolphinscheduler/extract/base/client/NettyRemotingClient.java

```diff
@@ -54,7 +56,8 @@ public class NettyRemotingClient implements AutoCloseable {
 
     private final Bootstrap bootstrap = new Bootstrap();
 
-    private final ConcurrentHashMap<Host, Channel> channels = new ConcurrentHashMap<>(128);
+    private final ReentrantLock channelsLock = new ReentrantLock();
```

Comment:
Is it better to use `ReentrantReadWriteLock`?

Reply:
This lock only used in write operation(change the channels map), the read operation will not use this lock.